browser-ui/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>wasm-python terminal</title>
+    <link rel="stylesheet" href="https://unpkg.com/xterm@4.16.0/css/xterm.css" crossorigin/>
+    <link rel="stylesheet" href="/styles.css"/>
+    <script src="https://unpkg.com/xterm@4.16.0/lib/xterm.js" crossorigin></script>
+    <script type="module" src="index.js"></script>
+</head>
+<body>
+    <textarea id="code"></textarea>
+    <div class="button-container">
+      <button id="run" disabled>Run</button>
+      <button id="repl" disabled>REPL</button>
+      <button id="clear" disabled>Clear</button>
+    </div>
+    <div id="terminal"></div>
+</body>
+</html>
```

Comment:
Missing newline at EOF
```suggestion
</html>

```